modules/transaction-payment/src/mock.rs

```diff
@@ -179,6 +179,9 @@ impl module_dex::Config for Runtime {
 parameter_types! {
 	pub MaxSwapSlippageCompareToOracle: Ratio = Ratio::saturating_from_rational(1, 2);
 	pub static TransactionByteFee: u128 = 1;
+	pub OperationalFeeMultiplier: u64 = 5;
+	pub TipPerWeightStep: u128 = 1;
```

Comment:
make it a higher value and have test to ensure tip value that is less than TipPerWeightStep results no priority change